src/Requests.php

```diff
@@ -621,7 +621,7 @@ public static function set_certificate_path($path) {
 	 * @param array|null $data Data to send either as a query string for GET/HEAD requests, or in the body for POST requests
 	 * @param string $type HTTP request type
 	 * @param array $options Options for the request
-	 * @return array $options
+	 * @return void
```

Comment:
While technically true, the `$options` parameter will be updated by reference, so I think if this is changed, that should be made clearer elsewhere in the docblock.